models/bulk_refund_request.go

```diff
@@ -0,0 +1,28 @@
+package models
+
+import "encoding/xml"
+
+// BatchService is the overall model that consists of all of the refunds
+type BatchService struct {
+	XMLName      xml.Name `xml:"batchService"`
+	Version      string   `xml:"version,attr"`
+	MerchantCode string   `xml:"merchantCode,attr" validate:"required"`
+	BatchCode    string   `xml:"batchCode,attr" validate:"required"`
+	Refunds      []Refund `xml:"refund" validate:"required,dive,required"`
+}
+
+// Refund is an individual worldpay refund
```

Comment:
If these structs are only for use with worldpay, then perhaps `BatchService` should be renamed to specify this.
Perhaps `WorldPayRefundBatch`?

Reply:
We're making the GovPay solution now not the WorldPay one. But agreed it could be named something like `GovPayRefundBatch`